.scripts/intercept.py

```diff
@@ -0,0 +1,42 @@
+# /// script
+# dependencies = [
+#     "polib",
+# ]
+# ///
+import argparse
+from pathlib import Path
+
+import polib
+
+
+def get_pofile_from_path(path: Path) -> polib.POFile:
+    if not path.exists():
+        raise ValueError(f"The path '{path.absolute()}' does not exist!")
+
+    if not (path.is_file() and path.suffix == ".po"):
+        raise ValueError(f"{path} doesn't seem to be a .po file")
+
+    try:
+        pofile = polib.pofile(path)
+    except OSError:
+        raise ValueError(f"{path} doesn't seem to be a .po file")
+    return pofile
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "path",
+        help="the path of a PO file",
+    )
+    parser.add_argument("-n", '--occurrence_number', type=int, default=1)
+    args = parser.parse_args()
+    path = Path(args.path).resolve()
```

Comment:
Add `type=Path`, and reformat to be more consistent:
```suggestion
    parser.add_argument("path", type=Path,
                        help="the path of a PO file")
    parser.add_argument("-n", '--occurrence_number',
                        type=int, default=1)
    args = parser.parse_args()
    path = args.path.resolve()
```
I would also add a `help=` for `-n`.